test/unstable-keyword/module/unstable_module.chpl

```diff
@@ -0,0 +1,18 @@
+//linear algebra
+//compilation error
+use LinearAlgebra;
+
+@unstable "The module 'Linear Algebra' is unstable" var A = Matrix(4,4, eltType=int);
```

Comment:
This test is a little misleading, since the module itself isn't what's being accessed and triggering the warning, but the warning mentions the module.  It might be worth changing the message so that it is clearer